app/assets/stylesheets/custom.css.scss

```diff
@@ -1,9 +1,46 @@
+/*-----------------------------------------
+Main Styles
+------------------------------------------*/
+
 body {
   background: image-url("pikachu_bg_opacity.png") no-repeat;
   background-size: cover;
   min-height: 100vh;
 }
 
+h2 {
+  font-size: 2.75rem !important;
+}
+
+h3 {
+  font-size: 2rem !important;
+}
+
+h4 {
+  font-size: 1.5rem !important;
+}
+
+h5 {
+  font-size: 1.20rem !important;
+}
+
+li {
+  list-style-type: none !important;
+}
+
+select {
+  display: block !important;
+}
+
+.team_member_index {
+  background: rgba(255, 255, 255, .5);
+  padding: 0px 20px 20px 20px !important;
+}
+
+/*-----------------------------------------
+Footer
+------------------------------------------*/
```

Comment:
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔

(Why not put these in a separate file?)